banyand/measure/service.go

```diff
@@ -98,6 +100,8 @@ func (s *service) Role() databasev1.Role {
 
 func (s *service) PreRun(_ context.Context) error {
 	s.l = logger.GetLogger(s.Name())
+	// set merge policy
+	customizeMergePolicy(withMaxFanOut(s.option.maxFanOutSize))
```

Comment:
You could get the `maxFanoutSize` from the `tst.option.maxFanOutSize`. You don't have to set it to a global var.